drift-transport-netty/src/main/java/com/facebook/drift/transport/netty/client/DriftNettyClientConfig.java

```diff
@@ -225,4 +231,55 @@ public DriftNettyClientConfig setCiphers(String ciphers)
                 .splitToList(requireNonNull(ciphers, "ciphers is null"));
         return this;
     }
+
+    public Boolean getConnectionPoolEnabled()
+    {
+        return connectionPoolEnabled;
+    }
+
+    @Config("thrift.client.connection-pool.enabled")
+    public DriftNettyClientConfig setConnectionPoolEnabled(Boolean connectionPoolEnabled)
+    {
+        this.connectionPoolEnabled = connectionPoolEnabled;
+        return this;
+    }
+
+    @Min(1)
+    public Integer getConnectionPoolMaxConnectionsPerDestination()
+    {
+        return connectionPoolMaxConnectionsPerDestination;
+    }
+
+    @Config("thrift.client.connection-pool.max-connections-per-destination")
+    public DriftNettyClientConfig setConnectionPoolMaxConnectionsPerDestination(Integer maxConnectionsPerDestination)
```

Comment:
We are also leaving these properties in ```DriftNettyConnectionFactoryConfig```. Is that mainly for backwards compatibility?

Reply:
Yeah

Reply:
Even though we haven't enabled [nativeTransport](https://github.com/prestodb/drift/blob/e55dade7a4884adb24a3b8d69c0bfe1c820fe2ed/drift-transport-netty/src/main/java/com/facebook/drift/transport/netty/client/DriftNettyConnectionFactoryConfig.java#L41) flag yet would it make sense to make it client specific as well.

Reply:
I was thinking about that. To make the transport client specific means we will need to maintain separate event listener for each, which means a separate thread pool for each client. I'm not sure if that is desired, as Ideally we don't want to spawn to many threads. 

Thoughts?

Reply:
@sujay-jain 

> We are also leaving these properties in DriftNettyConnectionFactoryConfig. Is that mainly for backwards compatibility?

Curious what does mean to have these properties both in `DriftNettyConnectionFactoryConfig` and `DriftNettyClientConfig`. Does that mean we can config it generally by `thrift.client.connection-pool.enabled` , but can also override by `<prefix>.thrift.client.connection-pool.enabled`?

Reply:
> . Does that mean we can config it generally by thrift.client.connection-pool.enabled , but can also override by <prefix>.thrift.client.connection-pool.enabled?

Yes